depot/lib/tram.ex

```diff
@@ -0,0 +1,84 @@
+defmodule Tram do
+  use GenServer
+
+  @moduledoc """
+  Попытка имплементации логики работы трамвая
```

Comment:
Не нужно никогда себя недооценивать. Скромничать уместно на воскресной мессе, в нашем ремесле надо быть уверенным, как танк :)
```suggestion
  Логика работы трамвая
```